src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Text.Json
+{
+    public enum JsonCommentHandling : byte
+    {
+        AllowComments = (byte)1,
+        Default = (byte)0,
+        SkipComments = (byte)2,
+    }
+    public partial class JsonReaderException : System.Exception
```

Comment:
JsonReaderException inherits some public virtual methods from Exception.  Should this be sealed?  Or override those methods as sealed?

Reply:
Yes, it should be sealed. Missed that. Thanks.

Reply:
Why seal? We are inconsistent - most of our exceptions are not sealed.

Reply:
In general I would like it if we sealed things by default, at least when there are virtuals, or at least sealed those overrides.  Not doing so constrains future improvements as we need to be concerned about derived behavior.  For exceptions, it's not as big a deal, but I still like the principle.

Reply:
Should it be called JsonSyntaxOptions? This would allow us to add more values in the future for controlling things other than comments.